src/common/sound.h

```diff
@@ -30,6 +30,21 @@ class Sound {
     double frequency;
     double volume;
 
+    uint32_t durations[eSOUND_TYPE_count];
```

Comment:
use reasonable data types and initialize them properly

Reply:
done